x-pack/legacy/plugins/index_management/index.ts

```diff
@@ -4,36 +4,26 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { resolve } from 'path';
-import { Legacy } from 'kibana';
-import { PLUGIN } from './common/constants';
-import { plugin as initServerPlugin, Dependencies } from './server';
-
-export type ServerFacade = Legacy.Server;
-
 export function indexManagement(kibana: any) {
```

Comment:
This file needs to stay until all dependent apps have migrated to the "plugins" folder.